css/style.css

```diff
@@ -94,3 +95,263 @@ header {
   padding: 20px 10px;
   margin: auto 0;
 }
+
+/* section works tonic1 */
+
+.btn {
+  color: #396df2;
+  box-sizing: border-box;
+  border-radius: 8px;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 18px;
+  line-height: 24px;
+  text-align: center;
+  letter-spacing: 0.05em;
+  padding: 15px;
+  margin: 0 10px;
+}
+
+.btn-enabled-tonic1 {
+  color: #6070ff;  /* #396DF2 */
+  background: #fff;
+  border: 1px solid #7f8cff;
+}
+
+.btn-enabled-tonic1:hover {
+  color: #fff;
+  background: #6070ff; /* * */
+  box-shadow: 0 8px 16px rgba(70, 85, 255, 0.25);
+}
+
+.btn-enabled-tonic1:active {
+  color: #fff;
+  background: #2230d2;
+}
+
+.btn-disable {
+  color: #5e6c84;
+  border: 1px solid #c1c7d0;
+}
+
+.tags-ul-tonic1 {
+  display: flex;
+  flex-flow: row nowrap;
+  gap: 10px;
+}
```

Comment:
- [ ] Please make sure to justify content left, right now the content is centered: 

Your Design | Figma Design
------------- | -------------
 ![image](https://user-images.githubusercontent.com/77038610/143585380-bf4b8ebe-cba4-4b8e-bd16-83e7a31e3a5a.png)|![image](https://user-images.githubusercontent.com/77038610/143585329-3c357714-1b9f-4a46-839d-cfd0560c43be.png)

You can simply do it by: 
```
.tags-ul-tonic1 {
    padding: 0; 
}
```